Tangem/Modules/Markets/TokenMarketsDetails/TokenMarketsDetailsViewModel.swift

```diff
@@ -388,13 +402,35 @@ private extension TokenMarketsDetailsViewModel {
                     return
                 }
 
-                Analytics.log(event: .marketsButtonAddToPortfolio, params: [.token: info.symbol])
+                Analytics.log(event: .marketsChartButtonAddToPortfolio, params: [.token: info.symbol])
 
                 coordinator?.openTokenSelector(with: info, walletDataProvider: walletDataProvider)
             }
         )
     }
 
+    func makeAnalyticsViewModel(using model: TokenMarketsDetailsModel) {
```

Comment:
тут не создается никакая вью модель, а отправляется аналитика. Вообще немного странная аналитика, т.к. некоторые блоки не везде присутствуют это считается ошибкой? Или у нас на старте будет по каждому токену инсайты?
```suggestion
    func sendBlocksAnalyticsErrors(using model: TokenMarketsDetailsModel) {
```

Reply:
> тут не создается никакая вью модель, а отправляется аналитика. Вообще немного странная аналитика, т.к. некоторые блоки не везде присутствуют это считается ошибкой? Или у нас на старте будет по каждому токену инсайты?

По названию согласен, не обратил внимание. По наполнению 
<img width="766" alt="Screenshot 2024-09-04 at 12 00 36" src="https://github.com/user-attachments/assets/328117de-844a-4def-a6ad-0640a02b559e">